src/Symfony/Component/AssetMapper/AssetMapperRepository.php

```diff
@@ -34,6 +34,7 @@ public function __construct(
         private readonly string $projectRootDir,
         private readonly array $excludedPathPatterns = [],
         private readonly bool $excludeDotFiles = true,
+        private readonly bool $debug = false
```

Comment:
I set this to true by default in aa469d9fb1

Reply:
When I did that locally while my project was set to `prod` mode, it causes Symfony to crash and return a 500 error.

<img width="608" alt="Screenshot of a 500 internal server error" src="https://github.com/symfony/symfony/assets/8751750/cb4881e7-651d-4b23-ac5e-5e18d8131e4f">

Is it possible that `param('kernel.debug'),` ([link to line](https://github.com/rynhndrcksn/symfony/blob/962a04450591a0c84bc1b5a0b0131217c406d907/src/Symfony/Bundle/FrameworkBundle/Resources/config/asset_mapper.php#L78)) isn't passing in `false` when the site is in `prod` mode, making `AssetMapperRepository.php` default to true, making it so the `$this->debug` checks succeed while the site is in `prod` mode?